smartcontactmanager/src/main/java/com/scm/smartcontactmanager/dao/category/CategoryDao.java

```diff
@@ -1,5 +1,33 @@
 package com.scm.smartcontactmanager.dao.category;
 
+import java.util.List;
+import java.util.Optional;
+
+import com.scm.smartcontactmanager.entity.category.Category;
+import com.scm.smartcontactmanager.entity.contact.Contact;
+import com.scm.smartcontactmanager.entity.contactcategoryrelation.ContactCategoryRelation;
+
 public interface CategoryDao {
 
+	public Category createCategory(Category category);
+
+	public List<Category> getListOfCategory();
+
+
+	Optional<Category> getCategoryById(int id);
+
+	List<Contact> getContactdByIds(List<Integer> contactIds);
+
+
+	List<ContactCategoryRelation> addContactsToCategory(List<ContactCategoryRelation> relations);
+
+	List<ContactCategoryRelation> getContactCategoryRelation(int id);
+
+	void deleteCategoryById(int id);
+	void deleteContactCategoryRelationById(List<Integer> collectId);
```

Comment:
Please follow proper indentation

Reply:
The issue have been resolved. Please check it once.